src/main/resources/templates/courseFormView.twig

```diff
@@ -1,31 +1,32 @@
-<div style="width:50%; margin: auto; padding-bottom: 60px;" class="collapse" id="courseForm">
+{% if courseCreationError %}<script>toastr.error("{{ courseCreationError }}")</script>{% endif %}
+<div style="width:50%; margin: auto; padding-bottom: 60px;" class="{% if not courseCreationError %}collapse{% endif %}" id="courseForm">
     <form action="/course" method="post">
         <div class="md-form">
-            <input placeholder="Course Name" type="text" name="name" class="form-control" id="courseFormName">
+            <input placeholder="Course Name" type="text" name="name" class="form-control" id="courseFormName"{% if courseCreationError %}value="{{ courseName }}"{% endif %}>
```

Comment:
Would we be able to pull in a js package so we don't have to manage the return values on error ourselves? Not sure if that's available